### PR TITLE
AT_11.01.01 | Build Executor Status |Create Node

### PR DESCRIPTION
**Pull request by EkBlinova**
https://trello.com/c/TvXNGnWi/2170-at110101-build-executor-status-create-a-new-node

### CODE REVIEW

File: cypress/pageObjects/HomePage.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ class HomePage {
     getgetDashboardMainPanel = () => cy.get('#main-panel');
     getProjectNameDrpDwnItems = () => cy.get('#breadcrumb-menu li');
     getFolderDrpDwnMenuList = () => cy.get('#breadcrumb-menu li[index]');
-    getBuildExecutorStatusLink = () => cy.get('a[href*="/computer/"]');
+    getBuildExecutorStatusLink = () => cy.get('div #executors .pane-header-title');
```
Comment (S-Lanchik) [inline cypress/pageObjects/HomePage.js:129]: Why did you change it? Use locator that already exist.

File: cypress/pageObjects/CreateItemPage.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+class CreateItemPage {
+    getSaveBtn = () => cy.get('button[name="Submit"]')
+
+    
+    clickSaveBtn() {
+        this.getSaveBtn().click();
+        return new NodesPage()
+    }
+}
+export default CreateItemPage;
```
Comment (S-Lanchik) [inline cypress/pageObjects/CreateItemPage.js:12]: I think it's better to name it NodeCreatePage

File: cypress/e2e/tests/buildExecutorStatus.cy.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        homePage
+        .clickBuildExecutorStatusLink() 
+        .clickNewNodeBtn()
+        .typeNodeNameInputField(newNodePageData.nodeName)
+        .selectPermanentAgentRadioBtn()
+        .clickCreateBtn()
+        .clickSaveBtn()
+        .verifyNewNodeExistsAndVisible(newNodePageData.nodeName)
+    })
+})
```
Comment (S-Lanchik) [inline cypress/e2e/tests/buildExecutorStatus.cy.js:21]: I would name this file like nodes.cy.js or nodesPage.cy.js

Comment (EkBlinova) [reply cypress/e2e/tests/buildExecutorStatus.cy.js]: Done

File: cypress/pageObjects/CreateItemPage.js (hunk 2 above)
Comment (EkBlinova) [reply cypress/pageObjects/CreateItemPage.js]: Done

File: cypress/pageObjects/HomePage.js (hunk 1 above)
Comment (EkBlinova) [reply cypress/pageObjects/HomePage.js]: It was not an exclusive locator, it needs to be changed

**Review by S-Lanchik — APPROVED** [review]

**Review by Rom-T — APPROVED** [review]
